packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -503,6 +503,177 @@
 
   <Shw::Divider />
 
+  <Shw::Text::H2>Layout</Shw::Text::H2>
+
+  <Shw::Text::H3>Interaction between table layout and columns width</Shw::Text::H3>
+
+  <Shw::Text::H4>Width in <code>px</code> + Table-layout = 'auto'</Shw::Text::H4>
+
+  <Hds::Table
+    @model={{this.model.manycolumns}}
+    @columns={{array
+      (hash key="first_name" label="First Name" isSortable="true" width="200px")
+      (hash key="last_name" label="Last Name" isSortable="true" width="200px")
+      (hash key="age" label="Age" isSortable="true")
+      (hash key="email" label="Email")
+      (hash key="bio" label="Biography" width="350px")
+    }}
+  >
+    <:body as |B|>
+      <B.Tr>
+        <B.Td>{{B.data.first_name}}</B.Td>
+        <B.Td>{{B.data.last_name}}</B.Td>
+        <B.Td>{{B.data.age}}</B.Td>
+        <B.Td>{{B.data.email}}</B.Td>
+        <B.Td>{{B.data.bio}}</B.Td>
+      </B.Tr>
+    </:body>
+  </Hds::Table>
+
+  <Shw::Text::H4>Width in <code>px</code> + Table-layout = 'fixed'</Shw::Text::H4>
```

Comment:
I know this example looks "broken" but I've left it on purpose to show what happens when a fixed layout is applied to the table, with only some columns having a width declared.

Reply:
(Maybe I should add a comment in the page to make this explicit?)